multi_model_experiment/04_novel_distance_calculations.py

```diff
@@ -308,23 +293,23 @@ def return_global_plot(year_model, tok="are", limits=(0, 1), inter_or_intra="int
 
 inter_model_files = list(Path("output/averaged_inter_models").rglob("*tsv"))
 unique_years = set(
-    list(map(lambda x: re.search(r"(\d+)", x.stem).groups()[0], inter_model_files))
+    list(map(lambda x: int(re.search(r"(\d+)", x.stem).groups()[0]), inter_model_files))
```

Comment:
I assume this is correct, but just wanted to double check that casting to an int is preferred over casting to a float here

Reply:
Yup. This is intentional as I kept running into type issues down stream. Prefer to keep years as ints.

Reply:
Ah I saw `inter_model_files` and thought this was weights; I totally missed that it was storing years :facepalm: